packages/terra-site/src/examples/arrange/ArrangeAlignment.jsx

```diff
@@ -32,12 +33,16 @@ class ArrangeAlignment extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      id: Math.floor(Math.random() * 0xFFFF),
+      id: undefined,
     };
-    this.getId = this.getId.bind(this);
+
     this.handleSelectChange = this.handleSelectChange.bind(this);
   }
 
+  componentWillMount() {
+    this.state.id = _.uniqueId();
+  }
+
   getId(name) {
     return name + this.state.id;
   }
```

Comment:
Should we update line 58 to set id={this.state.id}?

Reply:
Do you mean: `id={this.getId('align')}`?

If so, I don't think so. It seems to be using `getId()` to generate a related but unique identifier to be use in the form for the labels/select boxes.